calculator4.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
+    return x + y
+
+def subtract(x, y):
+    return x - y
+
+def multiply(x, y):
+    return x * y
+
+def divide(x, y):
+    if y == 0:
```

Comment:
Consider handling more edge cases, such as division by zero, in a user-friendly way. Instead of raising a ValueError, you could return a specific message or code. [medium]